discord/application_role_connection_metadata.go

```diff
@@ -0,0 +1,23 @@
+package discord
+
+type ApplicationRoleConnectionMetadata struct {
+	Type                     ApplicationRoleConnectionMetadataType `json:"type"`
+	Key                      string                                `json:"key"`
+	Name                     string                                `json:"name"`
+	NameLocalizations        map[Locale]string                     `json:"name_localizations,omitempty"`
+	Description              string                                `json:"description"`
+	DescriptionLocalizations map[Locale]string                     `json:"description_localizations,omitempty"`
+}
+
+type ApplicationRoleConnectionMetadataType int
+
+const (
+	IntegerLessThanOrEqual ApplicationRoleConnectionMetadataType = iota + 1
```

Comment:
for consistency sake you should prefix them with `ApplicationRoleConnectionMetadataType`. Great long consts 💀